app/containers/ItunesContainer/reducer.js

```diff
@@ -6,9 +6,10 @@ export const { Types: itunesContainerTypes, Creators: itunesContainerCreators }
   requestGetItuneTracks: ['trackName'],
   successGetItuneTracks: ['data'],
   failureGetItuneTracks: ['error'],
-  clearItuneTracks: {}
+  clearItuneTracks: {},
+  currentTrack: ['trackId']
```

Comment:
lets rename this to `setCurrentTrack` as this is an action, easier to understand

Reply:
sure, that would be a better naming.

Reply:
Please make the change